feat: add tests

yayauheny opened the pull request:
All changes:
- Add data, servuce, util packages into src/test/
- Add TestBuilder and other classes for testing functionality
- Total tests coverage is about 85 percent, should be finished by next commit

yayauheny committed:
feat: add tests

All changes:
- Add data, servuce, util packages into src/test/
- Add TestBuilder and other classes for testing functionality
- Total tests coverage is about 85 percent, should be finished by next
  commit

labanovichttps commented on src/test/java/com/console/ticket/service/ConsoleInputServiceTest.java:
```diff
@@ -0,0 +1,59 @@
+                .withCurrency(Currency.USA.getCurrency())
+                .build();
+        inputLine = "1-2 3-4 card-1111\nexit\n";
+//        currentTicketPath = Constants.DEFAULT_RECEIPT_PATH;
```
такого быть не должно

labanovichttps commented on src/test/java/com/console/ticket/service/ConsoleInputServiceTest.java:
```diff
@@ -0,0 +1,59 @@
+    @Mock
+    private static Company company;
+    private static String inputLine;
+//    private static Path currentTicketPath;
```
убрать комменты в коде

yayauheny committed:
refactor: delete unused lines

All changes:
- Deleted all unimplemented test classes
- Deleted all comment lines
